tests/api/test_api.py

```diff
@@ -23,16 +23,52 @@
     }
 }
 
+MOCK_DECISIONS_COLLECTION_RESPONSE = {
+    "user": {
+        "key": "{12345678-8901-2345-aaaa-bbbbbbcccccc}",
+    },
+    "decisions": {
+        "spocs": [mock_collection_response]
+    }
+}
+
+
 @responses.activate
-@schema.parametrize()
-def test_api(case: schemathesis.Case) -> None:
+@schema.parametrize(endpoint="/user")
+def test_delete_api(case: schemathesis.Case) -> None:
     # Mock call to forget API
     responses.delete("https://e-10250.adzerk.net/udb/10250/")
 
+    # Call Pocket Proxy and validate response
+    response = case.call_asgi()
+    case.validate_response(response)
+
+
+@responses.activate
+@schema.parametrize(endpoint="/spocs")
+def test_spocs_api(case: schemathesis.Case) -> None:
     with aioresponses() as m:
         # Mock call to decisions API
         m.post("https://e-10250.adzerk.net/api/v2", payload=MOCK_DECISIONS_RESPONSE)
 
         # Call Pocket Proxy and validate response
         response = case.call_asgi()
         case.validate_response(response)
+
+
+def version_two(context, body):
+    return body.get("version") == 2
+
+
+# clients pass in version=2 if they support collections
+@responses.activate
+@schema.hooks.apply(version_two, name="filter_body")
+@schema.parametrize(endpoint="/spocs")
+def test_spocs_collection_api(case: schemathesis.Case) -> None:
+    with aioresponses() as m:
+        # Mock call to decisions API
+        m.post("https://e-10250.adzerk.net/api/v2", payload=MOCK_DECISIONS_RESPONSE)
```

Comment:
Should this be `MOCK_DECISIONS_COLLECTION_RESPONSE`?

Reply:
good catch, i missed that from our debugging yesterday!